src/nvim/message.c

```diff
@@ -582,9 +582,24 @@ static bool emsg_multiline(const char *s, bool multiline)
         }
         redir_write(s, strlen(s));
       }
+
+      // Log (silent) errors as debug messages.
+      if (sourcing_name != NULL && sourcing_lnum != 0) {
+        DLOG("%s (%s (line %ld))", s, sourcing_name, (long)sourcing_lnum);
+      } else {
+        DLOG("%s", s);
```

Comment:
these should have some sort of indication that they were silenced.
```suggestion
        DLOG("(:silent) %s", s);
```